README.md

```diff
@@ -1,68 +1,102 @@
 
-### Task 2: MVP
+* [ ] Next, we need to be able to load in the current movie's attributes into our local form state. Add in the code needed to retrieve the movie data of the passed in id when the component mounts.
 
-#### Updating A Movie:
+* [ ] At this point, nothing happens when the edit form is submitted. Add in the service call need to update the server with our updated movie when the edit form is submit button is clicked.
 
-- Add a route at the path `/update-movie/:id`
-- Create a component with a form to update the chosen movie
-- Add a button in the movie component that routes you to your new route with the movies's id as the URL param
-- The form should make a PUT request to the server when submitted
-- When the call comes back successfully, reset your form state and route the user to `/` where they will see the updated movie in the list
-- (Editing the movie stars can be a stretch problem... don't worry about it now. Move on to the next step, and come back and solve this at the end)
+* [ ] Don't forget to make sure that your server data and your local state are in sync! Make any changes the edit route needed to modify global state within the edit form.
 
-Movie object format:
+* [ ] Now that we have access to setMovies, made sure the updated list of movies is saved to our global state.
 
-```
-{
-  id: 5,
-  title: 'Tombstone',
-  director: 'George P. Cosmatos',
-  metascore: 89,
-  stars: ['Kurt Russell', 'Bill Paxton', 'Sam Elliot'],
-}
-```
+* [ ] Redirect the user to the currently edited movie's indivisual info page.
 
-#### Deleting A Movie:
+### Deleting a Movie
+> *You added in a CRUD feature! Good job! Now let's get deleted squared away...*
 
-- Add a delete button in the movie component that makes a DELETE request
-- When the call comes back successfully, route the user to `/` where they will see the updated movie list without the deleted movie
+* [ ] Identify the component that holds the "button" needed for deletion. Add an event hanlder to that button.
 
-#### Adding A Movie:
+* [ ] Build your new event handler to make the request needed to delete the currentlt viewed movie. Observe what is returned from the request.
 
-- Add a route at the path `/add-movie`
-- Create a component with a form to add a new movie
-- Each created movie should have the following format (notice the array of strings - this will test your JS skills, so work through it methodically)
-- The form should make a POST request to the server when submitted
-- When the call comes back successfully, reset your form state and route the user to `/`
+* [ ] You will once again need to keep the server and state data in sync. In the `App.js` class, complete complete the deleteMovie function so that it recieves a movie id and sets movies to be equal to all movies BUT the movie with the indicated id.
 
-Movie object format:
+* [ ] Make the changes needed to give the correct component addes to your deleteMovie function.
```

Comment:
Check this sentence. I'm not sure what's being asked. The word "addes" is confusing here 🤔